js/layout-javascript/news-feed-code.js

```diff
@@ -483,7 +545,18 @@ DynamicList.prototype.attachObservers = function() {
     .on('hide.bs.collapse', '.news-feed-filters-panel .panel-collapse', function() {
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
-    .on('click', '.news-feed-comment-holder', function(event) {
+    .on('keydown', '.news-feed-filters-panel', function(event) {
```

Comment:
Why does this not listen to the `click` event like all other listeners?

```suggestion
    .on('click keydown', '.news-feed-filters-panel', function(event) {
```

Reply:
@tonytlwu hi, i tried to do a squash commit and this code was lost, now fixed.